init.sh

```diff
@@ -40,31 +40,52 @@ installPreCommit() {
 }
 
 while true; do
-  read -p "Do you want to install precommit? (Y/n) " yn
+  read -p "Do you want to install pre-commit? (Y/n) " yn
```

Comment:
Maybe rename it to python and pre-commit, i forgot about it too.

Reply:
Noted.